Source/RunActivity/Viewer3D/Popups/TrainDrivingWindow.cs

```diff
@@ -822,8 +832,13 @@ void AddLabel(ListLabel label)
                 }
                 else
                 {
-                    index = trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("BC"));
+                    if (trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("V"), index) > 0)
+                        index = trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("V"), index);
+                    else
+                        index = trainBrakeStatus.IndexOf(Viewer.Catalog.GetString("BC"));
+
                     brakeInfoValue = trainBrakeStatus.Substring(index, trainBrakeStatus.Length - index).TrimEnd();
+                    brakeInfoValue = brakeInfoValue.StartsWith(Viewer.Catalog.GetString("V")) ? brakeInfoValue.Replace(Viewer.Catalog.GetString("V"), Viewer.Catalog.GetString("V") + "  ") : brakeInfoValue;
```

Comment:
I just have a question: what's the use of adding those spaces to the string ?

Reply:
The brake system tokens (**GetTrainBrakeStatus())** use two characters (BC, BP, EQ…), the "V" looks not aligned, compared to the rest, without adding those spaces to the string.

Reply:
Ok, thank you.